docs/misc/operation/aws-ami.md

```diff
@@ -2,6 +2,8 @@
 
 Kaia offers an AWS AMI ([Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)) for Kaia endpoint nodes (EN). These AMIs are pre-installed with both the EN software and the chaindata, enabling users to quickly and conveniently launch a fully operational node. With Kaia's AMI, setting up a new endpoint node on demand becomes seamless, allowing for simplifying the process of joining the Kaia network.
 
+> Note: The Kaia Foundation provided AWS AMIs for the Kairos testnet until October 29, 2024, and will cease to provide these after November 1, 2024
```

Comment:
Do we need to mention twice? the one below the table seems enough.

Reply:
Tried to make sure readers do not miss it.
Updated!